federation/index.md

```diff
@@ -44,6 +44,30 @@ but is not yet recommended for processing medium size to large countries or cont
 
 The collection metadata of the Terrascope back-end tries to clearly identify which collections are served by SentinelHub.
 
+#### Commercial Data
+
+openEO Platform provides direct access to commercial data. Currently, data must be purchased directly through Sentinel Hub (see Sentinel Hub documentation on purchasing commercial data [here](https://docs.sentinel-hub.com/api/latest/api/data-import/)), but we are working to support ordering commercial data directly from the platform in the future. 
+
+Data is accessed as part of the load_collection process and via a 'featureflag' argument. To access the data, you must:
```

Comment:
@dthiex Is it featureflag or featureflags? Here it's mixed up in several places.

Reply:
I think it should be featuresflags everywhere, corrected.